src/help/h_tags.php

```diff
@@ -22,10 +22,7 @@ function print_intro() {
             $conflictmsg = " and conflicted PC members";
         }
 
-        echo "<p>PC members and administrators can attach tag names to papers.
-It’s easy to change tags and to list all papers with a given tag,
-and <em>ordered</em> tags preserve a particular paper order.
-Tags also affect color highlighting in paper lists.</p>
+        echo "<p>PC members and administrators can attach tag names to applicaitons. It’s easy to change tags and to list all applications with a given tag, and <em>ordered</em> tags preserve a particular application order. Tags also affect color highlighting in application lists.</p>
```

Comment:
Could you not change the line breaks but just the text? again easier for future merge.